frontend/src/screens/learner.tsx

```diff
@@ -64,236 +103,226 @@ const LearnerDashboard = () => {
             <HighlightedStudies />
 
             <StudiesContainer />
+            {isVisible && (
+                <Box
```

Comment:
@divitcr7 did you not want to use the [affix component?](https://mantine.dev/core/affix/)